File: Basic Coding Problems/02 If - Elses/026 Write a program to check whether a character is in the alphabet or not/C/check_alphabet.c
#include <stdio.h>

int main() {
    char character;

    printf("Enter a character: ");
    scanf("%c", &character);

    if (character >= 'a' && character <= 'z' || character >= 'A' && character <= 'Z') {
        printf("%c is in the alphabet.\n", character);
    } else {
        printf("%c is not in the alphabet.\n", character);
    }

    return 0;
}

File: Basic Coding Problems/02 If - Elses/026 Write a program to check whether a character is in the alphabet or not/Java/check_alphabet.java
import java.util.Scanner;

public class check_alphabet {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a character: ");
        char character = scanner.next().charAt(0);

        boolean isLetter = Character.isLetter(character);

        if (isLetter) {
            System.out.println(character + " is in the alphabet.");
        } else {
            System.out.println(character + " is not in the alphabet.");
        }
    }
}